### ISSUE TITLE
Version 50+ having trouble picking up initial notifications.

### ISSUE THREAD

**Issue by djryan** [simple-last-fm-scrobbler/sls#503]
There was on scrobble in the cache, which got pushed to last.fm, but it can't pick up any more.  I've uninstaller, reset caches and removed local data, but it's still not noticing that anything new has been played (to push to the now playing api) or has been played.

**Comment by a93h** [comment]
Well that's really not good.

**Comment by a93h** [comment]
This is my apk or the playstore version?

**Comment by djryan** [comment]
Double checked and it seems to be both.
Migjt be something in the app detection code; I can't get it to pick up new apps either.

**Comment by a93h** [comment]
Strange.
Have you tried Exiting the app? I added an exit button on the initial screen.
Sometimes this helps.
This is really bad news.
I did not change any  of the code there.

**Comment by a93h** [comment]
Ok I recreated your bug.
There is an app clash problem upon initial install I believe.
If I play apps that previously used Broadcast intents it clashes.

Do you happen to know if all the apps you have been trying use broadcast intents in the old way?

**Comment by a93h** [comment]
I looked at Androids example mediamanager external player app and found out the real issue.
Thanks so much for pointing this issue out.

**Comment by a93h** [comment]
#508

**Comment by WolfFhang** [comment]
Moving conversation from #507 to here like I was told earlier.

After trying 1.6.2 using multiple media players (Youtube, Play Music, GMMP) SLS still doesn't pick up any of the apps in the "Enabled Apps" section, nor can I get it to scrobble. Not sure if others are still having issues on 1.6.2


Also a potentially unrelated Issue that might need a separate thread, the stats from the older version (1.5.7 in my case) are still missing in the last several updates (1.6.0-1.6.2 etc), yet if I revert to version 1.5.7 the Stats show up fine and are still there

**Comment by a93h** [comment]
Yes, stats are not typically kept through versions.
I will add that to my todo list.

**Comment by a93h** [comment]
However, version 1.6.2 should pick up new music players now.
Did you enable the Notification Listener AKA Notification Access permissions?

**Comment by WolfFhang** [comment]
Ah it seems I was mistaken, I went and made sure I had that permission enabled and it was off/not allowed. After allowing that Notification permission it seems as though at least the Now Playing portion is working, but songs aren't scrobbling once finished.


I was a bit confused because I saw we were now allowed to skip the permission if we didn't want to enable them all. I was under the assumption that the Notification Access was only needed if you used the Active App Notification option
